floodlight/models/kinematics.py

```diff
@@ -0,0 +1,329 @@
+import numpy as np
+
+from floodlight import XY
+from floodlight.core.property import PlayerProperty
+from floodlight.models.base import BaseModel
+
+
+class DistanceModel(BaseModel):
+    """Computations for euclidean distances of all players.
+
+    Upon calling the :func:`~DistanceModel.fit`-method, this model calculates the
+    framewise euclidean distance for each player. The following calculations can
+    subsequently be queried by calling the corresponding methods:
+
+        - Frame-wise Distance Covered --> :func:`~DistanceModel.distance_covered`
+        - Cumulative Distance Covered -->
+        :func:`~DistanceModel.cumulative_distance_covered`
+
+    Notes
+    -----
+    The model calculates the distance in the unit of the corresponding position data.
+    This may lead to distortions when used in combination with models that use SI units
```

Comment:
"This may lead to distortions when used in combination with models that use SI units" - I don't quite understand this statement, is there a typo? Or do you want to say that for metrical units, the output equals the input unit? If so, I'd just state it that way